test/tic_tac_toe_test.exs

```diff
@@ -4,4 +4,9 @@ defmodule TicTacToeTest do
   test "passes the test automatically" do
     assert true
   end
+
+  @tag :skip
+  test "fails the test automatically" do
+    assert false
+  end
```

Comment:
You can remove this now if you want. I can look through the history of CircleCI jobs to see that the failing test failed the workflow https://app.circleci.com/pipelines/github/jonmanahan/Tic-Tac-Toe?branch=circle_ci&filter=all

Reply:
Sounds good to me, Ill remove that and push it to this branch! Would that be ready to merge after?